jdaviz/core/helpers.py

```diff
@@ -336,13 +336,16 @@ def show(self, loc="inline", title=None):  # pragma: no cover
 
             NOTE: Only applicable to a "sidecar" display.
 
+        height: int, optional
+            The height of the top-level application widget, in pixels.
```

Comment:
should we mention that this applies to all (inline) instances of the same app?

Reply:
I never open multiple instances, so I didn't think of this. I'll clarify.